storage_service/common/management/commands/create_aip_replicas.py

```diff
@@ -141,18 +143,15 @@ def _delete_replicas(self, aip_uuid):
         return deleted_count
 
     def _delete_replica(self, replica):
-        """Delete replica from filesystem and database
+        """Delete replica from filesystem and update Package status
+
+        If filesystem deletion is successful, the replica's status will be
+        set to Package.DELETED by `replica.delete_from_storage()`.
```

Comment:
Given the context of this second PR I understand this comment. I wonder if this reflects a need to improve the documentation of the function in the storage service code itself... :thinking:

Reply:
I am in no hurry, as this version works for me (unless I'm missing something important). 
Please proceed as you see fit.

Reply:
@amayita not at all, running this version from the branch seems like a good approach to me. Thanks for the response!

Reply:
@ross-spencer I think in this case the method docstring was clear enough, and the fault was mine for not reading closely.